### PR TITLE
Make the header sticky

### PR DESCRIPTION
**Pull request by stefankarlberg**
# [PT Story](https://www.pivotaltracker.com/story/show/166565659)

##  Changes we propose in this PR:
* Makes the header sticky on scroll.

## What we learned:
* How to make a sticky header without using Semantic UI.

### CODE REVIEW

File: src/CSS/Header.css [hunk 1]
```diff
@@ -5,4 +5,10 @@
 .header_selector {
   border: none !important;
   margin: 2px !important;
-}
\ No newline at end of file
+}
+
+.stickyClass {
```
Comment (tochman) [inline src/CSS/Header.css]: Please do not use camelCase in css class names.

Comment (raptorf1) [reply src/CSS/Header.css]: changed to snake case. thanks.

**Comment by tochman** [comment]
I’m not sure about the convention, but shouldn’t you keep the /build folder from VCS?

**Comment by raptorf1** [comment]
> I’m not sure about the convention, but shouldn’t you keep the /build folder from VCS?

You are right. By default the `build` folder is in the `gitignore` file. But sometimes Netlify gets stuck during building and then you have to `yarn build` locally and deploy to Netlify with no build command. That's the case here. Netlify gave us an error when we installed the react-redux-token-auth, so from that point on we build locally and then deploy.

**Pull request merged**